exercises/two-bucket/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+/// A struct to hold your results in.
+#[derive(PartialEq, Eq, Debug)]
+pub struct BucketStats<'a> {
+    /// The total number of "moves" it should take to reach the desired number of liters, including
+    /// the first fill.
+    pub moves: u8,
+    /// Which bucket should end up with the desired number of liters? (Either "one" or "two")
+    pub goal_bucket: &'a str,
```

Comment:
Let's leverage Rust's capabilities, and use a two-element `enum` to represent the final bucket. It's more type-safe, and a better example for learners.